docs/data/material/components/alert/alert.md

```diff
@@ -8,81 +8,155 @@ waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/alert/
 
-## Variants
+#### Filled
 
-Two additional variants are available – outlined, and filled:
+{{"demo": "FilledAlerts.js"}}
 
-### Outlined
+#### Outlined
 
 {{"demo": "OutlinedAlerts.js"}}
 
-When using an outlined alert with the [`Snackbar` component](/material-ui/react-snackbar/#customization), background content will be visible and bleed through the alert by default.
-You can prevent this by adding `bgcolor: 'background.paper'` to the[`sx` prop](/material-ui/customization/how-to-customize/#the-sx-prop) on the `Alert` component.
+:::warning
+When using an outlined Alert with the [Snackbar](/material-ui/react-snackbar/) component, background content will be visible and bleed through the Alert by default.
+You can prevent this by adding `bgcolor: 'background.paper'` to [the `sx` prop](/material-ui/customization/how-to-customize/#the-sx-prop) on the Alert component:
 
-### Filled
+```jsx
+<Alert sx={{ bgcolor: 'background.paper' }} />
+```
 
-{{"demo": "FilledAlerts.js"}}
+Check out the [Snackbar—customization](/material-ui/react-snackbar/#customization) doc for an example of how to use these two components together.
+:::
+
+### Actions
 
-## Toast
+Add an action to your Alert with the `action` prop.
+This lets you insert any element—an HTML tag, an SVG icon, or a React component such as a Material UI Button—after the Alert's message, justified to the right.
+
+If you provide an `onClose` callback to the Alert without setting the `action` prop, the component will display a close icon (&#x2715;) by default.
+
+{{"demo": "ActionAlerts.js"}}
 
-You can use the Snackbar to [display a toast](/material-ui/react-snackbar/#customization) with the Alert.
+### Icons
 
-## Color
+Use the `icon` prop to override an Alert's icon.
+As with the [`action`](#actions) prop, your `icon` can be an HTML element, an SVG icon, or a React component.
+Set this prop to `false` to remove the icon altogether.
 
-The `color` prop will override the default color for the specified severity.
+If you need to override all instances of an icon for a given [`severity`](#basics), you can use the `iconMapping` prop instead.
+You can define this prop globally by customizing your app's theme. See [Theme components—Default props](/material-ui/customization/theme-components/#default-props) for details.
+
+{{"demo": "IconAlerts.js"}}
+
+### Titles
+
+To add a title to an Alert, import the Alert Title component:
+
+```jsx
+import AlertTitle from '@mui/material/AlertTitle';
+```
+
+You can nest this component above the message in your Alert for a neatly styled and properly aligned title, as shown below:
+
+{{"demo": "DescriptionAlerts.js"}}
+
+### Color
+
+Use the `color` prop to override the default color for the specified [`severity`](#basics)—for instance, to apply `warning` colors to a `success` Alert:
 
 {{"demo": "ColorAlerts.js"}}
 
-## Accessibility
+### Transitions
```

Comment:
I'm not sure it's customization. I mean, where does customization ends? Since "Overriding the root slot" is 't part of it, I would imagine it's about visual design

```suggestion
## Transitions
```

Reply:
Good point about the Transitions section. As for "Overriding the root slot," I'm torn about that decision. I feel that the info is more closely related to the "Anatomy" section, but it would also make sense under "Customization."

Reply:
@oliviertassinari another option would be to move the "Overriding" info further down the page, maybe to a new section called "Advanced" below "Customization"? Since I assume the majority of users are not going to bother with it.

Reply:
> to move the "Overriding" info further down the page, maybe to a new section called "Advanced" below "Customization"

I would remove this `Overriding the root slot` section altogether. It sounds like something all the components support, so not worth to be duplicate on each component guide page.

Reply:
So then, don't mention this info at all on the Material UI component pages?

Reply:
Yeah, and maybe only cover this in a customization guide 🤷‍♂️

Reply:
Hmm, well then before I move forward with more component page revisions like this one, I'll write that customization guide and then we can decide how it should connect with these pages.

Reply:
For example in https://mui.com/base/react-select/#slot-props, we are describing the API of the `SelectUnstyled` component. It feels like something that would be better in the reference docs: https://mui.com/base/api/select-unstyled/.